### DISCUSSION TITLE
Asset Acceptance

### DISCUSSION THREAD

**Discussion by cookierun22** [snipe/snipe-it#11973] [Q&A] [answered]
Hi can you help me on how to set that user able to receive notification when assign asset and able to do the physical signature to accept the asset? I dont know how to configure for this option. Thank you.

**Comment by ndrach02** [comment]
I have the same question, have you been able to figure it out?

**Comment by Bigrazorstorm** [comment]
I would also need the function. I would like to issue some assets directly in the office and not have to spend ages chasing confirmation.

**Reply by Bigrazorstorm** [reply]
It would also be great if you could generate a receipt via PDF when returning the item, as employees may have little mailboxes left at that time (offboarding)

**Comment by swift2512** [comment, marked as answer]
Do you use Snipe-IT at all? :D Literally, there's  a checkbox **Require users to confirm acceptance of assets in this category.** when creating/editing asset category. With this enabled, user receives an email asking to accept assets and also is being shown message that he has unaccepted items, when logged in.
![image](https://github.com/user-attachments/assets/f5bd0141-919a-41da-a3b7-aeaf2979fe23)

From there, it's either accept/decline with a push of a button or by scribbling the signature if enabled in admin settings.

![image](https://github.com/user-attachments/assets/70c4d71a-13c9-420f-bcd3-49117ed19caf)

If you want 'pen and paper' approach, print out list of items assigned to particular user and get his signature.

![image](https://github.com/user-attachments/assets/e573a335-4a55-45b7-a834-bb01919f2b64)
